src/libxrpl/protocol/Indexes.cpp

```diff
@@ -313,6 +315,22 @@ depositPreauth(AccountID const& owner, AccountID const& preauthorized) noexcept
         indexHash(LedgerNameSpace::DEPOSIT_PREAUTH, owner, preauthorized)};
 }
 
+// Credentials should be sorted here, use credentials::makeSorted
```

Comment:
This comment is redundant - `std::set` is always sorted

Reply:
I asked for this comment, as a note for possible future refactors

Reply:
It is review request